community.html

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Community
+---
+
+{% include nav.html active='Community' %}
+
+<section class="front-section">
+  <div class="container">
+    <div class="col-wide">
+
+      <h1>Community</h1>
+
+      <p>
+        The following resources are available for you if you have a question regarding Neovim:
+      </p>
+
+      <ul>
+        <li>Chat: <a href="https://gitter.im/neovim/neovim">Neovim on Gitter</a></li>
+        <li>Mailing list: <a href="https://groups.google.com/forum/#!forum/neovim">Neovim Google Group</a></li>
+        <li>IRC: <a href="https://webchat.freenode.net/?channels=neovim"><code>#neovim</code> on Freenode</a></li>
+        <li>Reddit: <a href="https://reddit.com/r/neovim">/r/neovim</a></li>
+        <li><a href="https://vi.stackexchange.com">vi StackExchange</a> (use the <code>neovim</code> tag)</li>
+      </ul>
+
+      <p>
+      If you want to report a bug, please use <a href="https://github.com/neovim/neovim/issues">Neovim's issue tracker on Github</a>.
```

Comment:
`Github` should be `GitHub`, and perhaps instead of:
```
If you want to report a bug, ...
```
it could be
```
If you want to report a bug or suggestion, ...
```
or more generally
```
If you want to report an issue, ...
```